### PR TITLE
Broker CI Pipeline Template Integration (Plus Helpful droidbranch Command)

### PR DESCRIPTION
**Pull request by fadidurah**
This PR covers work done to introduce templates to the azure-pipelines/ui folder in android-complete, mainly to be used in broker-test.yml. This work will make creating more UI Automation pipelines in android-complete easier, as well as making a very large pipeline yml file much easier to understand by breaking it up into smaller files.

Broker CI Run with this change where build steps are passing: https://dev.azure.com/IdentityDivision/Engineering/_build/results?buildId=1004655&view=results

Also added a .gitconfig command that I added to my local branch a while ago, I felt it could be useful to other people on the team. All it does is print the current branch of each of the submodules. It's not necessary to this PR.

### CODE REVIEW

File: azure-pipelines/ui-automation/templates/run-on-firebase.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      - script: gcloud version
+        displayName: 'Check gcloud version'
+      - task: Bash@3
+        name: runUiAutomation
+        displayName: Run UI Automation on Firebase
+        inputs:
+          targetType: inline
+          script: |
+            gcloud auth activate-service-account --key-file "$(gcServiceAccountKey.secureFilePath)"
+            gcloud config set project "$(gCloudProjectId)"
```
Comment (p3dr0rv) [inline azure-pipelines/ui-automation/templates/run-on-firebase.yml:46]: where this come from?

**Review by p3dr0rv — APPROVED** [review]

**Comment by p3dr0rv** [comment]
the run-on-firebase yamls, are tightly coupled to the broker-test pipeline, primarily because of the other-files option, in the following work we can separate this.
Thanks Fadi.

File: azure-pipelines/ui-automation/templates/run-on-firebase.yml (hunk 1 above)
Comment (fadidurah) [reply azure-pipelines/ui-automation/templates/run-on-firebase.yml:46]: This is a pipeline variable

**Comment by fadidurah** [comment]
> the run-on-firebase yamls, are tightly coupled to the broker-test.yml, primarily because of the --other-files option, in the following work we can separate this. Thanks Fadi.

Will address this as part of the next PR

**Pull request merged**